### ISSUE TITLE
Show error/warning messages on elements that don't accept ::before and ::after

### ISSUE THREAD

**Issue by jackdomleo7** [jackdomleo7/Checka11y.css#52]
### Describe the new a11y feature or project enhancement

A solution to showing error/warning on void HTML elements such as `<img />` or elements that cannot have a `::before` and `::after` attribute such as `<iframe>`. We currently show a border with no error/warning text. The following elements are ones I've found so far that don't accept the pseudo-elements (`::before` and `::after`):
- `<img>`
- `<iframe>`
- `<audio>`
- `<select>`
- `<object>`
- `<input>`
- `<textarea>`
- `<video>`
- And more I haven;t discovered, but it covers all void elements and some others.

### Describe the solution you'd like

Any that works accessibly and correctly.

### Link(s)

- https://www.w3.org/TR/2011/WD-html-markup-20110113/syntax.html#void-elements
- https://jsfiddle.net/YCZh6/1/
- https://stackoverflow.com/questions/3538506/which-elements-support-the-before-and-after-pseudo-elements
- https://www.w3.org/TR/CSS21/generate.html

**Comment by cat-street** [comment]
Just a random thought: set an background with error message picture URL, position it at the bottom and set a padding-bottom for it to be visible.
![Снимок экрана 2020-10-04 233621](https://user-images.githubusercontent.com/2076436/95026572-dd274280-069a-11eb-8c3d-e0bfd5c89360.png)

**Comment by jackdomleo7** [comment]
Anyone picking this item up, there is a solution in #65 that just needs to be applied to elements without a warning/error. Would be good to see some good developer documentation on how to use this warning/error and when to use it as opposed to the `content` property.
